### DISCUSSION TITLE
In order to use Copilot, I need to log into GitHub whenever I restarted IntelliJ IDEA

### DISCUSSION THREAD

**Discussion by 0x7FFFFFFFFFFFFFFF** [community/community#8074] [Copilot Conversations] [answered]
Hi,

Whenever I restart IntelliJ IDEA, I have to relog into GitHub in order to to use Copilot. Is is expected? Thanks.

**Comment by jansorg** [comment, marked as answer]
No, it's not expected. One of the next updates will fix this.
Are you on macOS? Some macOS systems seem to always deny the access to the keychain, where the GitHub info is stored.
Please verify that passwords are saved at `Preferences > Appearance & Behavior > System Settings > Passwords`. You could also try "KeePass" as a workaround, until this is fixed.

**Reply by 0x7FFFFFFFFFFFFFFF** [reply]
Thanks for the clarification. I'm on Windows. I'll wait for  the fix.
